staticfiles/templates/contributors/pitkofsky.html

```diff
@@ -0,0 +1,54 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Josh Pitkofsky</title>
+</head>
+<body>
+<div class="row text-center" ng-controller="HomeController as vm">
+  
+  <div class="row">
+  	<div class="center col-sm-12"> 
+  	   <p>
+  	   </p>
+  	</div>
+  </div>
+
+  <div class="row">
+    <div class="col-md-12">
+
+      <div>
+        <div class="row">
+          <div class="col-lg-12">
+              <h1 class="page-header"><small>Hi, I'm Josh</small></h1>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-sm-3 col-xs-6">
+            <img class="thumbnail img-responsive" src="/static/images/contributors/pitkofsky.jpg" />
+          </div>
+          <div class="col-sm-9 col-xs-6">
+            <blockquote>
+                Sophomore at <a target="_blank" href="https://www.carleton.edu/">Carleton College</a>.<br> Computer Science Major.<br> <a target="_blank" href="https://github.com/AdamCanady/Neuro/blob/master/README.md"> Neuroscience </a>concentrator. <br> <a target="_blank" href="http://www.acm.edu/our_collaborations/Nick_Adams_Short_Story_Contest.html">Writer</a>. Film <a href="https://vimeo.com/user39115933">Fanatic</a>.</h2>
+				<p>Here is <a target="_blank" href="https://youtu.be/n9KPaCdtTAA">a video </a>about a hackathon I helped organize.
+            </blockquote>
+            
+            <blockquote>
+                <a href="https://www.linkedin.com/in/joshpitkofsky" target="_blank" class="pull-left">LinkedIn</a>
+                <br />
+            </blockquote>
+
+            <br />
+            <br />
+            <br />
+          </div>
+
+        </div>
+
+        </div>
+    </div>
+</body>
+</html>
```

Comment:
routers and home html are missing. Also test the code in local environment and fix few errors associated with pitkofsky.html